File: CSharp/Tests/RectanglePerimeterTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class RectanglePerimeterTest
    {
        [Theory]
        [InlineData(6, 7, 26)]
        [InlineData(20, 10, 60)]
        [InlineData(2, 9, 22)]
        public void FindPerimeter_IntLengthAndWidthValues_ReturnIntPerimeter(int length, int width, int expected)
        {
            var actual = RectanglePerimeter.FindPerimeter(length, width);

            Assert.Equal(expected, actual);
        }
    }
}

File: js/tests/rectanglePerimeter.test.js
const findPerimeter = require("../rectanglePerimeter");

const data = [
  [6, 7, 26],
  [20, 10, 60],
  [2, 9, 22]
];

describe("findPerimeter", () => {
  describe("Return rectangle perimeter from given length and width", () => {
    test.each(data)("2(%i) + 2(%i) = %i", (length, width, perimeter) => {
      var actual = findPerimeter(length, width);

      expect(actual).toEqual(perimeter);
    });
  });
});
